cli/inagoctl.go

```diff
@@ -34,6 +38,12 @@ var (
 			// This callback is executed after flags are parsed and before any
 			// command runs.
 
+			loggingConfig := logging.DefaultConfig()
+			if globalFlags.Verbose {
+				loggingConfig.LogLevel = "DEBUG"
+			}
```

Comment:
Minor and not your fault. Here we see that required features do not match functionality of our internal implementations. Such Feature flips and jolty redirects often indicate some misconceptions.